server/app/views/admin/AdminLayout.html

```diff
@@ -0,0 +1,79 @@
+<!--/*@thymesVar id="templateGlobals" type="views.admin.shared.TemplateGlobals"*/-->
+<!--/*@thymesVar id="layoutParams" type="views.admin.shared.LayoutParams"*/-->
+<!doctype html>
+<html lang="en-US" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <title
+      th:text="${#strings.isEmpty(templateGlobals.pageTitle()) ? 'CiviForm' : templateGlobals.pageTitle() + ' - CiviForm'}"
+    >
+    </title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="civiform-build-tag"
+      th:content="${layoutParams.civiformImageTag()}"
+    />
+    <meta
+      th:if="${layoutParams.addNoIndexMetaTag()}"
+      name="robots"
+      content="noindex"
+    />
+    <meta
+      name="htmx-config"
+      th:content='|{"inlineStyleNonce":"${templateGlobals.cspNonce()}"}|'
+    />
+
+    <link rel="icon" th:href="${layoutParams.favicon()}" />
+
+    <!--/* Stylesheets to be added */-->
+    <link
+      th:each="stylesheet: ${layoutParams.stylesheets()}"
+      th:nonce="${templateGlobals.cspNonce()}"
+      th:href="${stylesheet}"
+      type="text/css"
+      rel="stylesheet"
+    />
+    <!--/* Scripts to be added before body content. Keep these to a minimum  */-->
+    <script
+      th:each="script: ${layoutParams.headScripts()}"
+      th:nonce="${templateGlobals.cspNonce()}"
+      th:src="${script}"
+      type="text/javascript"
+    ></script>
+  </head>
+  <body class="display-flex minh-viewport body-background-gradient">
+    <!--/* Header navigation */-->
+    <th:block th:insert="~{admin/shared/AdminCommonHeader}"></th:block>
+
+    <!--/* Main */-->
+    <main
+      class="padding-x-4 padding-y-10 bg-white shadow-2 width-full maxw-widescreen margin-x-auto"
+    >
+      <h1
+        th:text="${#strings.isEmpty(templateGlobals.pageTitle()) ? 'Page Title Missing' : templateGlobals.pageTitle()}"
+      ></h1>
+      <th:block
+        th:insert="~{${layoutParams.pageTemplate()} :: content}"
+      ></th:block>
+    </main>
+
+    <!--/* Google Tag Manager */-->
+    <th:block
+      th:insert="~{components/GoogleTagManagerFragment :: tagManager(${layoutParams.measurementId()}, ${templateGlobals.cspNonce()})}"
+    ></th:block>
+
+    <!--/* Scripts to be added at the end */-->
+    <script
+      th:each="script: ${layoutParams.bodyScripts()}"
+      th:nonce="${templateGlobals.cspNonce()}"
+      th:src="${script}"
+      type="text/javascript"
+      defer
+    ></script>
+
+    <!--/* Optional page specific scripts. These should be load last. */-->
+    <th:block
+      th:insert="~{${layoutParams.pageTemplate()} :: scripts} ?: ~{}"
```

Comment:
First, apparently `?: ~{}` is called the Elvis operator. That seemed important to share.

Second, The inclusion of it here is a fallback if the `scripts` fragment is undefined in the specified `pageTemplate()`.